scripts/readme_translations.py

```
@@ -0,0 +1,211 @@
+        WAYS: "शुरू करने के तीन तरीके। कोई एक चुनें।",
+        LICENSE_LINE: "MIT। जैसे चाहें इस्तेमाल करें: फ़ोर्क करें, पढ़ाएँ, बेचें, प्रकाशित करें। श्रेय देना अच्छा है, पर ज़रूरी नहीं।",
+        MAINTAINED: "[Rohit Ghumare](https://github.com/rohitg00) और समुदाय द्वारा अनुरक्षित।",
+        H_HOW: "यह कैसे काम करता है", H_CURR: "पाठ्यक्रम की संरचना", H_LESSON: "एक पाठ की संरचना",
+        H_START: "शुरुआत करें", H_PREREQ: "आवश्यक शर्तें", H_BOOK: "इसे किताब की तरह पढ़ें",
+        H_SHIPS: "हर पाठ कुछ न कुछ देता है", H_CONTENTS: "विषय-सूची", H_TOOLKIT: "टूलकिट",
+        H_WHERE: "कहाँ से शुरू करें", H_WHY: "यह अभी क्यों मायने रखता है", H_CONTRIB: "योगदान करें",
+        H_SPONSOR: "काम को प्रायोजित करें", H_STAR: "स्टार इतिहास", H_LICENSE: "लाइसेंस",
+    },
+    "ar": {
+        HERO1: "**‏84% من الطلاب يستخدمون أدوات الذكاء الاصطناعي بالفعل، لكن 18% فقط يشعرون بأنهم مستعدون لاستخدامها باحتراف.** هذا المنهج يسدّ هذه الفجوة.",
+        HERO2: "‏503 دروس. 20 مرحلة. نحو 320 ساعة. Python وTypeScript وRust وJulia. كل درس ينتج مخرجًا قابلًا لإعادة الاستخدام: موجّهًا، أو مهارة، أو وكيلًا، أو خادم MCP. مجاني، مفتوح المصدر، برخصة MIT.",
```

Ruff (0.16.0)

[error] Contains control characters that can permit obfuscated code

(PLE2502)